addons/website/views/snippets/s_cards_grid.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cards_grid" name="Cards Grid">
+    <section class="s_cards_grid o_cc o_cc2 pt64 pb64 o_colored_level" data-vxml="001" data-vcss="001">
+        <div class="container">
+            <div class="col-lg-12">
+                <h3>Features that set us apart</h3>
+            </div>
+            <div class="row d-flex align-items-stretch">
+                <div data-name="Card" class="col-lg-6">
```

Comment:
As the card is used alone inside the column, we have to transfer the editor options of the card to it's parent column (so it's not duplicated and it does not conflict)

You can check how I did [here](https://github.com/odoo/odoo/pull/172768/files#diff-9f41c68ba854940dda3361dc32795e2d7e9fc5b83ef7f8afd35a455b493e9930R573) If you have questions, DM me